sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp

```diff
@@ -32,7 +32,7 @@ namespace Azure { namespace Storage {
     {
     }
 
-    void SetAccountKey(std::string accountKey)
+    void UpdateAccountKey(std::string accountKey)
```

Comment:
I am assuming docs will be coming later for some of these?

Reply:
yeah, we're making a lot of changes. I'm going to sum up and make up all the changelog before next release.

Reply:
That works for the changelog.

I meant adding API doxygen docs (brief, param, return, exceptions, etc.).

Reply:
Well, I have no idea why it doesn't have doxygen docs.

Reply:
i'm going to add it right now